spec/spec.md

```diff
@@ -503,6 +503,7 @@ get_capabilities
     "supports_sign_and_send_transactions": <supports_sign_and_send_transactions>,
     "max_transactions_per_request": <max_transactions_per_request>,
     "max_messages_per_request": <max_messages_per_request>,
+    "supported_transaction_versions": [<supported_transaction_versions>, ...]
```

Comment:
nit / not sure we should do anything about this, but the other fields use `supports_` and this one uses `supported_`